app/controllers/api/proposals_controller.rb

```diff
@@ -34,8 +35,19 @@ def show
     render json: @proposal
   end
 
+  def update
```

Comment:
Oh, I see. Maybe we should have a special permisson, like:

```
can? :review, Proposal
```

Would make a bit more sense to me.